client/layout/sidebar/menu.jsx

```diff
@@ -0,0 +1,10 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+const SidebarMenu = ( { children } ) => (
+	<li className="sidebar__menu">{ children }</li>
```

Comment:
> I'm going to finish this one once I create a centralized `sidebar/item.jsx`.

You might be able to reduce some of `client/my-sites/sidebar/sidebar-menu-item.jsx`.